### PR TITLE
Improve loading states docs

### PR DESCRIPTION
**Pull request by anilmaurya**
https://github.com/emberjs/guides/issues/1352

### CODE REVIEW

File: source/localizable/routing/loading-and-error-substates.md [hunk 1]
```diff
@@ -57,19 +57,28 @@ Router.map(function() {
...
 syntax is acceptable. This can be useful for creating a custom loading screen
 for a leaf route like `slow-model`.
 
+When accessing `foo.bar` route then Ember will search for:
+
+1. `foo.bar-loading`
+2. `foo.loading` or `foo-loading`
+3. `loading` or `application-loading`
+
+It's important to note that `foo.bar.loading` is not consider now.
```
Comment (acorncom) [inline source/localizable/routing/loading-and-error-substates.md]: is not considered

Comment (anilmaurya) [inline source/localizable/routing/loading-and-error-substates.md]: Ok, will update.

**Comment by toddjordan** [comment]
@homu r+

**Comment by toddjordan** [comment]
Thanks @anilmaurya !

**Pull request merged**